docs/people-and-groups/saml-keycloak.md

```diff
@@ -35,7 +35,10 @@ For more information, check out our guide for [authenticating with SAML](./authe
 7. Configure the service provider (Metabase) from **Configure** > **Realm Settings**.
     - From **Endpoints**, select “SAML 2.0 Identity Provider Metadata”.
     - An XML file will open in a new tab.
-8. From the XML file, note the following:
+8. Configure the SLO service
```

Comment:
This is optional, right? Maybe we should note that

Reply:
cc @jeff-bruemmer for wording

Reply:
Yeah, you only need to set that up if you want Single Logout to work.